samples/arm-none-eabi/reported.by

```diff
@@ -0,0 +1,310 @@
+reporter_name="Jasmin Jessich"
+reporter_url="http://jasmin.anw.at/"
+reporter_comment="NOTE: Please read samples/arm-none-eabi/reported.by prior to any built!"
+
+#
+# This configuration will build an arm-none-eabi compiler to be used for
+# embedded development. The original build script can be found here:
+#    https://launchpad.net/gcc-arm-embedded
+# To see the original build script, you have to download the installer:
+#    https://launchpad.net/gcc-arm-embedded/5.0/5-2015-q4-major
+# 
+# I tried to mimic the original configure options and environment settings with
+# crosstool-ng (CT_NG) as close as possible. I really analysed every option
+# which was not provided by CT-NG or which was added. The description in this
+# file explains each diversion from the original.
+# 
+# NOTE:   newlib-nano is currently not built!
+# 
+# The original uses the git/SVN versions of binutils-gdb, gcc and newlib. CT-NG
+# doesn't currently support using a git or SVN repository for the source
+# location. In fact this is not really necessary, because the official packages
+# contain most of the required code already.
+# Only armv8-m is not available in the official packages. This core is very new
+# and even not available on GCC trunk, so I guess we can wait for the next GCC
+# release to add it.
+#
+# If someone really want to experiment with the mentioned core variant, feel
+# free to use them as custom location in CT-NG.
```

Comment:
> This was ABSOLUTELY NOT my intention! I simply forgot the required 
> patches :(
> THX for your review!
>
Ah, that explains the failure :) Thanks for fixing it!
Will try it out again tonight when I get home.

Regards,
Alexey.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/crosstool-ng/crosstool-ng/pull/372/files/9ea84294c84ab9ba765e8aeafa3648c36edc1f28#r56924434>
>